openedx/features/course_card/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+
+
+class CourseCard(models.Model):
+    """
+    These courses are used for representing course re-runs.
```

Comment:
Can we update this line as "These courses are used for representing course classes".

Reply:
This name looks more appropriate in code since these are re-runs as depicted by model name as well.